src/dos/program_autotype.h

```diff
@@ -0,0 +1,17 @@
+#include "programs.h"
```

Comment:
Hey, this file will be a good candidate to test clang-format ;)

Reply:
Worked perfectly!